tests/sessions_tests/tests.py

```diff
@@ -713,3 +714,36 @@ def test_unpickling_exception(self):
 
         self.session.serializer = PickleSerializer
         self.session.load()
+
+
+class SessionSettingsTests(unittest.TestCase):
+
+    @override_system_checks([check_session_settings])
```

Comment:
Take a look at the pattern used in `tests/check_framework/test_templates.py` -- that doesn't require `@override_system_checks` and is a bit cleaner I think.